homeassistant/components/lifx/light.py

```diff
@@ -13,6 +14,7 @@
 import voluptuous as vol
 
 from homeassistant import util
+from homeassistant.components import network
```

Comment:
Add `network` as a dep in `manifest.json`

`  "dependencies": ["network"],`

Reply:
I'm fairly sure I did.